### DISCUSSION TITLE
Old PHP and Laravel version

### DISCUSSION THREAD

**Discussion by quangvhk** [cjmellor/level-up#41] [Ideas]
I know Laravel 10 is good. But there are still someone who using Laravel 6 like me. Could you make an old version for this nice thing? Or tell me if I can bridge it myself. Thanks!

**Comment by cjmellor** [comment]
Hi,

I won't be making this compatible with anything other than the current supported versions.

If you want to use this with an older version of Laravel, may I suggest forking the repo and changing the minimum Laravel version and then just making the required changes where needed? There will be some trial and error but you should be able to get it working with some persistence.

Thanks.
